requirements.txt

```diff
@@ -14,3 +14,4 @@ flake8-pyi==20.5.0
 mccabe
 pycodestyle==2.6.0
 pyflakes==2.2.0
+https://download.pytorch.org/whl/cpu/torchaudio-0.10.0%2Bcpu-cp36-cp36m-linux_x86_64.whl
```

Comment:
torchaudio is installed by conda, so it is not required here.

Reply:
our SDK environment is using python3.6, and the installation steps may have to follow the IPU_README.md.

there is also a problem for the checking below: `UnitTest / unit-test (ubuntu-latest, 3.8, 1.10.0, 0.10.0) (pull_request) Failing after 2m — unit-test (ubuntu-latest, 3.8, 1.10.0, 0.10.0)`